mitodl/mitxonline
# Update eslint-config and friends

### What are the relevant tickets?
Opened this to check
- https://github.com/mitodl/eslint-config/pull/47

### Description (What does it do?)
Updates our eslint config from `eslint-config-mitodl` v0.7 to v2+. See comments on eslintrc.


### How can this be tested?
1. Reinstall yarn dependencies (`docker compose restart watch` or `yarn install`)
2. run `yarn workspace mitx-online-public lint` it should pass
3. run `yarn workspace mitx-online-public fmt-check` it should pass
4. You could also try triggering an error, e.g., an unused constant

### NOTE:

~⚠️  This is using a git-installed version of eslint-config-mitodl. After that version is published, this PR should be updated.~
